### PR TITLE
Hyacinth 652 previous next on digital object show

### PR DESCRIPTION
**Pull request by JackBlackLight**

### CODE REVIEW

File: app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js [hunk 1]
```diff
@@ -1,22 +1,70 @@
...
+    pageNumber: undefined,
+    perPage: undefined,
+    filters: undefined,
+    orderBy: undefined,
+    ...JSON.parse(latestSearchQueryString),
+  };
+
+  const {
+    q, filters, orderBy,
+  } = decodeQueryParams(queryParamsConfig, queryParams);
```
Comment (elohanlon) [inline app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js:39]:
```suggestion
  } = decodeQueryParams(queryParamsConfig, queryParams);
  
  const [orderField, orderDirection] = orderBy.split(' ');
```

Maybe good to double check my syntax locally, but I think the above destructuring assignment should work.  This suggestion is related to another suggestion below.

File: app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js [hunk 2]
```diff
@@ -1,22 +1,70 @@
...
+
+  const {
+    loading, error, data,
+  } = useQuery(
+    getDigitalObjectIDsQuery, {
+      variables: {
+        limit,
+        offset: Number(offset),
+        searchParams,
+        orderBy: { field: orderBy.split(' ')[0], direction: orderBy.split(' ')[1] },
```
Comment (elohanlon) [inline app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js:51]:
```suggestion
        orderBy: { field: orderField, direction: orderDirection },
```
See earlier comment for declaration of these two variables.

Comment (JackBlackLight) [reply app/javascript/hyacinth_v1/components/digital_objects/DigitalObjectInterface.js:51]: I tried these changes out locally and they worked well.

File: app/javascript/hyacinth_v1/components/shared/ResultsPagingBar.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        onClick={() => onResultClick(offset - 1, resultIndex - 1, uidPrev)}
+        disabled={resultIndex === 1}
+      />
+      <>
+        <Pagination.Item active>
+          {resultIndex}
+          {' '}
+            of
+          {' '}
+          {totalCount}
```
Comment (elohanlon) [inline app/javascript/hyacinth_v1/components/shared/ResultsPagingBar.js:38]:
```suggestion
          {`${resultIndex} of ${totalCount}`}
```
Slightly more compact alternative.

**Review by elohanlon — APPROVED** [review]

**Pull request merged**